app/models/abstract_contributor.rb

```diff
@@ -78,17 +76,4 @@ def self.grouped_roles(citable:)
   def person?
     contributor_type == "person"
   end
-
-  # used by DraftWorkForm
-
-  def role_term
-    [contributor_type, role].join(SEPARATOR)
-  end
-
-  # used by DraftWorkForm for setting values from the form.
-
-  def role_term=(val)
-    contributor_type, role = val.split(SEPARATOR)
-    self.attributes = {contributor_type:, role:}
-  end
 end
```

Comment:
don't need this abstraction anymore, we can set the `role` and `contributor_type` directly from the drop down menu and radio buttons